po/fr.po

```diff
@@ -8,16 +8,16 @@ msgid ""
 msgstr ""
 "Project-Id-Version: bazaar\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2025-07-07 22:10+0000\n"
-"PO-Revision-Date: 2025-07-08 00:14+0200\n"
-"Last-Translator: Atom <29772841+AtomHare@users.noreply.github.com>\n"
+"POT-Creation-Date: 2025-07-08 14:12+0200\n"
+"PO-Revision-Date: 2025-07-08 15:13+0200\n"
+"Last-Translator: Léane GRASSER <leane.grasser@proton.me>\n"
 "Language-Team: French\n"
 "Language: fr\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n > 1);\n"
```

Comment:
I think this line needs to be updated because it creates newlines starting with "", I'm not familiar enough with localization editors to know if it's something to take care of or not